docs/iceberg-catalog.md

```diff
@@ -11,63 +11,86 @@ import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
 ## Introduction
 
-DataHub Iceberg Catalog provides integration with Apache Iceberg, allowing you to manage Iceberg tables through DataHub. This tutorial walks through setting up and using the DataHub Iceberg Catalog to create, read, and manage Iceberg tables. The catalog supports bi-directional sync of metadata and provides a secure way to manage Iceberg tables through DataHub's permissions model.
+DataHub Iceberg Catalog provides integration with Apache Iceberg, allowing you to manage Iceberg tables through DataHub. This tutorial walks through setting up and using the DataHub Iceberg Catalog to create, read, and manage Iceberg tables. The catalog supports provides a secure way to manage Iceberg tables through DataHub's permissions model  while also enabling discovery use-cases for humans instantly.
```

Comment:
“The catalog supports provides a secure way…” -> “The catalog provides a secure way…”